server/controllers/event.controller.js

```diff
@@ -18,12 +18,12 @@ function validateEventType(req, res, next) {
 
 function newEvent(req, res, next) {
   const { eventType } = req.body;
-  producer.sendMessage(eventType, JSON.stringify(req.body), (error, eventId) => {
+  influxInterface.write(eventType, req.body, (error) => {
```

Comment:
@andrewmarklloyd  What is the reason for not  publishing the even stream via the event stream producer and sending it directly to influx db?

Reply:
@andrewmarklloyd  ignore. Just saw Jeff's comment on slack.